tools/webpack/shared.js

```diff
@@ -64,13 +64,15 @@ const plugins = [
 	process.env.WP_BUNDLE_ANALYZER && new BundleAnalyzerPlugin(),
 	new DefinePlugin( {
 		// Inject the `IS_GUTENBERG_PLUGIN` global, used for feature flagging.
-		'process.env.IS_GUTENBERG_PLUGIN':
-			process.env.npm_package_config_IS_GUTENBERG_PLUGIN,
+		'globalThis.IS_GUTENBERG_PLUGIN': JSON.stringify(
```

Comment:
I like the direction. We will have to remember to update the WordPress Core so tree-shaking continues to work.

Reply:
Adding the "php backport" label. Although it's not really PHP :)

Reply:
What is to be backported? Is this file shared with core?

Reply:
The webpack config change.

Reply:
It would be a very small and simple (but important!) change. I'll open a PR for that soon so it's ready.

Reply:
I've opened https://core.trac.wordpress.org/ticket/61262 and https://github.com/WordPress/wordpress-develop/pull/6596.